keyv/Map.h

```diff
@@ -136,7 +136,11 @@ class Map
      * @version 1.9.2
      */
     template< class V > bool insert( const std::string& key, const V& value )
-        { return _insert( key, value, std::is_trivially_copyable< V >( ));}
+#if __GNUG__ && __GNUC__ < 5
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc-3.3.3/cpp/Common-Predefined-Macros.html

__GNUG__
    The GNU C++ compiler defines this. Testing it is equivalent to testing (__GNUC__ && __cplusplus).